nowcasting_dataset/filesystem/utils.py

```diff
@@ -136,12 +157,20 @@ def upload_one_file(
         local_filename: the local file name
 
     """
-    filesystem = fsspec.open(remote_filename).fs
+    filesystem = get_filesystem(remote_filename)
     filesystem.put(local_filename, remote_filename)
 
 
-def make_folder(path: Union[str, Path]):
-    """ Make folder """
-    filesystem = fsspec.open(path).fs
-    if not filesystem.exists(path):
-        filesystem.mkdir(path)
+def makedirs(path: Union[str, Path], exist_ok: bool = True) -> None:
```

Comment:
This new code should be funcationally equivalent.

Renaming to `makedirs` to be consistent with `fsspec.FileSystem.makedirs`